packages/grafana-data/src/types/data.ts

```diff
@@ -38,6 +38,12 @@ export type PreferredVisualisationType = typeof preferredVisualizationTypes[numb
 export interface QueryResultMeta {
   type?: DataFrameType;
 
+  /**
+   * TypeVersion is the version of the Type property. Versions greater than 0.0 correspond to the dataplane
+   * contract documentation https://github.com/grafana/grafana-plugin-sdk-go/tree/main/data/contract_docs.
+   */
+  typeVersion?: string;
```

Comment:
Currently this is `[2]uint` in go. (For major minor). Perhaps we want to match this better here? I could possibly change the backend to an object with name uint props as well... Not sure what suits typescript and our patterns here. @ryantxu happen to have an opinion on this?

Reply:
In https://github.com/grafana/grafana-plugin-sdk-go/pull/597/files#r1084458472 I suggest we just use a semver string.  That seems like the simplest approach

But FYI in typescript, this could be defined as:
```
typeVersion?: [number,number];
```